### PR TITLE
Fixes #549

### LINKED ISSUE #549: TenantActions#getOrganizations(OrganizationCriteria)'s javadoc is wrong
The javadoc for `getOrganizations(OrganizationCriteria)` in `TenantActions` refers to `Directories` rather than `Organizations`

### PR DESCRIPTION
**Pull request by jarias**
JavaDoc fix

Fixes #549

### CODE REVIEW

File: api/src/main/java/com/stormpath/sdk/tenant/TenantActions.java [hunk 1]
```diff
@@ -227,16 +230,15 @@
...
      * the criteria DSL.  For example:
      * <pre>
-     * client.getDirectories(Directories
-     *     .where(Directories.description().containsIgnoreCase("foo"))
-     *     .and(Directories.name().startsWithIgnoreCase("bar"))
-     *     .orderByName().descending()
-     *     .withAccounts(10, 10)
-     *     .offsetBy(20)
-     *     .limitTo(25));
+     * client.getDirectories(Organizations
```
Comment (lhazlewood) [inline api/src/main/java/com/stormpath/sdk/tenant/TenantActions.java]: still references `getDirectories` when it should be `getOrganizations` ;)

Comment (dogeared) [inline api/src/main/java/com/stormpath/sdk/tenant/TenantActions.java]: In my PR, there's this:

https://github.com/stormpath/stormpath-sdk-java/pull/544/files#diff-98a4a3bb69930369909e2949cd96cb23R204

Looks like you still need to call `client.getOrganizations` here

Comment (jarias) [inline api/src/main/java/com/stormpath/sdk/tenant/TenantActions.java]: LOL how did I miss that

File: api/src/main/java/com/stormpath/sdk/tenant/TenantActions.java [hunk 2]
```diff
@@ -227,20 +230,19 @@
 
     /**
      * Returns a paginated list of the current tenant's organizations that match the specified query criteria.  The
-     * {@link com.stormpath.sdk.directory.Directories Directories} utility class is available to help construct
+     * {@link com.stormpath.sdk.organization.Organization Organization} utility class is available to help construct
```
Comment (mrioan) [inline api/src/main/java/com/stormpath/sdk/tenant/TenantActions.java]: ..organization.Organizations Organizartions

**Comment by mrioan** [comment]
@jarias back to you. If you agree with the proposed change you can fix it and consider this PR approved. Thanks

**Pull request merged**